1-starter/README.md

```diff
@@ -36,18 +36,60 @@
   </p>
 </div>
 
-# 📜 Documentation
+## 🏗 Development
 
-> **[discord-ts.js.org](https://discord-ts.js.org)**
+```
+npm install
+npm run dev
+```
 
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+If you want to use [Nodemon](https://nodemon.io/) to auto-reload while in development:
 
-# ☎️ Need help?
+```
+npm run watch
+```
+
+## 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+### 🐋 Docker (Recommended)
+
+To start your application:
+
+```
+docker-compose up -d
+```
+
+To shut down your application:
+
+```
+docker-compose down
+```
+
+To view your application's logs:
+
+```
+docker-compose logs
+```
+
+For the full command list please view the [Docker Documentation](https://docs.docker.com/engine/reference/commandline/cli/).
+
+## 📜 Documentation
+
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
+
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Join and ask in our [Discord](https://discord-ts.js.org/discord)
 
-# Thank you
+## 💖 Thank you 💖
 
-Show your support for [discordx](https://www.npmjs.com/package/discordx) by giving us a star on [github](https://github.com/oceanroleplay/discord.ts).
+If you feel like supporting us in making [Discordx](https://www.npmjs.com/package/discordx), Why not give us a star on [GitHub](https://github.com/oceanroleplay/discord.ts)?
```

Comment:
I would like this to revert

Reply:
Ye but new one sounds less stiff

Reply:
I agree with @TeamWolfyta, this new one sounds less forceful

Reply:
I agree, we may need more polite line, but I am not agree on "feel"

Reply:
I can think of something better then, what about `Why not support us in making DiscordX by giving us a star on Github`

Reply:
revert for now, make another PR for this, but I still not happy with new suggestion.